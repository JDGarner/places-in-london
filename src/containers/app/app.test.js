import React from "react";
import App from "./app";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

let component;

beforeEach(() => {
  component = shallow(<App />);
});

test("<App /> renders correctly", () => {
  expect(toJson(component)).toMatchSnapshot();
});

test("Clicking on next place button should increment currentPlaceIndex", () => {
  expect(component.instance().state.currentPlaceIndex).toEqual(0);
  component.find("button").simulate("click");
  expect(component.instance().state.currentPlaceIndex).toEqual(1);
});
