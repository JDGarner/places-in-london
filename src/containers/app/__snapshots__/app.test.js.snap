// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
<div>
  <header
    className="header"
  >
    <h2>
      Stuff I Like
    </h2>
  </header>
  <Place
    name="Nomadic Community Gardens"
  />
  <footer>
    <button
      onClick={[Function]}
    >
      Next Place
    </button>
  </footer>
</div>
`;
